project/app/static/webpack_sources/js/ho_dan.js

```diff
@@ -1,7 +1,16 @@
 import 'bootstrap';
 import $ from 'jquery';
+import Choices from 'choices.js'
+
 
 $(document).ready(() => {
+
+    // Init the choices.js instances :
+    let selectTinh = new Choices(document.getElementById('select-tinh'));
+    let selectHuyen = new Choices(document.getElementById('select-huyen'));
+    let selectXa = new Choices(document.getElementById('select-xa'));
+    let selectStatus = new Choices(document.getElementById('select-status'));
```

Comment:
In the issue, there is no requirement to add a search to select status and just a few items, we don't need search for status.